webapp/channels/src/i18n/en.json

```diff
@@ -3285,10 +3285,10 @@
   "cloud.fetch_error.retry": "Retry",
   "cloud.invoice_pdf_preview.download": "<downloadLink>Download</downloadLink> this page for your records",
   "collapsed_reply_threads_modal.confirm": "Got it",
-  "collapsed_reply_threads_modal.description": "Threads have been revamped to help you create organized conversation around specific messages. Now, channels will appear less cluttered as replies are collapsed under the original message, and all the conversations you're following are available in your **Threads** view. Take the tour to see what's new.",
   "collapsed_reply_threads_modal.skip_tour": "Skip Tour",
   "collapsed_reply_threads_modal.take_the_tour": "Take the Tour",
   "collapsed_reply_threads_modal.title": "A new way to view and follow threads",
+  "collapsedReplyThreadsModal.helpText": "Threads have been revamped to help you create organized conversation around specific messages. Now, channels will appear less cluttered as replies are collapsed under the original message, and all the conversations you're following are available in your <b>Threads</b> view. Take the tour to see what's new.",
```

Comment:
@M-ZubairAhmed, Should we follow the `snake_case` rather than `CamelCase`? I know this has nothing to do with impact, but it's just for consistency across the file.

Reply:
i have queued this item for discussion in Web guild to see what others think about this